app/controllers/file_controller.rb

```diff
@@ -50,7 +50,9 @@ def show
     disposition = params[:disposition] || 'inline'
     filename = params["filename"] || "#{objid}#{fileid}"
     headers['Content-Disposition'] = "#{disposition}; filename=#{filename}"
-    if ds.mimeType
+    if filename.include?('.jpg')
```

Comment:
@mcritchlow The example file bb0478952m_2.jpg in https://github.com/ucsdlib/damspas/issues/325 doesn't get corrupted. It displays fine after I open it.   
The MIME is associated with record metadata, but I think the file itself doesn't contain it.

There are two options to get the Calisphere harvest working:
1. Fixing it in the program as what I did. I think Calisphere doesn't harvest metadata, does it?
2. Working with Ops to search the DAMS database and find all octet-stream for jpeg, and fix the metadata.

Which option do you prefer?


Do you prefer to update